src/pages/home/report/ReportActionItem.js

```diff
@@ -150,58 +150,72 @@ class ReportActionItem extends Component {
-                        <View>
-                            {this.props.shouldDisplayNewIndicator && (
-                                <UnreadActionIndicator />
+            <Hoverable resetsOnClickOutside>
+                {hovered => (
+                    <View>
+                        {this.props.shouldDisplayNewIndicator && (
+                            <UnreadActionIndicator />
+                        )}
+                        <View
+                            style={StyleUtils.getReportActionItemStyle(
+                                hovered
+                                || this.state.isContextMenuActive
+                                || this.props.draftMessage,
                             )}
-                            <View
-                                style={StyleUtils.getReportActionItemStyle(
-                                    hovered
-                                    || this.state.isContextMenuActive
-                                    || this.props.draftMessage,
-                                )}
-                            >
-                                {!this.props.displayAsGroup
-                                    ? (
-                                        <ReportActionItemSingle action={this.props.action} showHeader={!this.props.draftMessage}>
+                        >
+                            {!this.props.displayAsGroup
+                                ? (
+                                    <ReportActionItemSingle action={this.props.action} showHeader={!this.props.draftMessage}>
+                                        <PressableWithSecondaryInteraction
+                                            ref={el => this.popoverAnchor = el}
+                                            onPressIn={() => this.props.isSmallScreenWidth && canUseTouchScreen() && ControlSelection.block()}
+                                            onPressOut={() => ControlSelection.unblock()}
+                                            onSecondaryInteraction={this.showPopover}
+                                            preventDefaultContentMenu={!this.props.draftMessage}
+                                            onKeyDown={(event) => {
+                                                // Blur the input after a key is pressed to keep the blue focus border from appearing
+                                                event.target.blur();
+                                            }}
+                                        >
                                             {children}
-                                        </ReportActionItemSingle>
-                                    )
-                                    : (
+                                        </PressableWithSecondaryInteraction>
+                                    </ReportActionItemSingle>
+                                )
+                                : (
+                                    <PressableWithSecondaryInteraction
```

Comment:
@mollfpr same `PressableWithSecondaryInteraction` is repeated twice, we should be able to DRY it up right?